packages/launchpad/src/components/topnav/TopNavListItem.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <li
+    class="flex transition-colors duration-300 border-b border-b-gray-50"
+    :class="selectable ? 'hover:bg-indigo-50' : ''"
+  >
+    <div v-if="hasPrefix">
+      <div class="flex items-center h-full align-middle">
+        <slot name="prefix" />
+      </div>
+    </div>
+    <div class="flex-grow">
+      <slot />
+    </div>
+    <div>
+      <div
+        v-if="hasSuffix"
+        class="flex items-center h-full align-middle"
+      >
+        <slot name="suffix" />
+      </div>
+    </div>
+  </li>
+</template>
+<script setup lang="ts">
+import { useSlots, computed } from 'vue'
+const slots = useSlots()
+
+const hasPrefix = computed(() => {
+  return Boolean(slots.prefix)
+})
+
+const hasSuffix = computed(() => {
+  return Boolean(slots.suffix)
+})
```

Comment:
You can get at slots in the template and remove all of this code.

`v-if="$slots.suffix"`

Reply:
There we go. I grabbed this from some other place we are doing it, but I see that we only did it there because we're using the output elsewhere in the script block.